curriculum/challenges/english/25-front-end-development/lab-blog-post-card/66eaddd04a9e533fba689001.md

```diff
@@ -0,0 +1,393 @@
+---
+id: 66eaddd04a9e533fba689001
+title: Design a Blog Post Card
+challengeType: 14
+dashedName: lab-blog-post-card
+demoType: onClick
+---
+
+# --description--
+
+In this lab, you'll practice how to style backgrounds and borders by creating a blog post card.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should have a `div` with a class of `blog-post-card` to hold all your card elements.
+2. You should have an image with a valid `alt` text and the class of `post-img`. You can use `https://cdn.freecodecamp.org/curriculum/labs/cover-photo.jpg` for the `src` attribute of your image.
+3. You should have a `div` with a class of `post-content`.  
+   Within the `.post-content` `div`:
+
+   - You should have a `h2` with a class of `post-title`. Assign some text to act as the title of your blog post.
+   - You should have a `p` with a class of `post-excerpt`. Assign some text to summarize the blog post (maximum of 20 words).
+   - You should have an `a` with a class of `read-more`. Assign the text `Read More` to it.
+
+4. You should apply the following styles to the `blog-post-card` div:
+
+   - A white colored background.
+   - Rounded corners.
+   - Shadow for depth.
+   - Appropriate `overflow` property to prevent content from overflowing.
```

Comment:
Likewise, I don't think `overflow` is taught prior to this. We remove this and the test if not.